manifests/a/ArtificialQualia/PyEveLiveDPS/2.7.2/ArtificialQualia.PyEveLiveDPS.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ArtificialQualia.PyEveLiveDPS
+PackageVersion: 2.7.2
+Installers:
+- InstallerUrl: https://github.com/ArtificialQualia/PyEveLiveDPS/releases/download/v2.7.2/PELD-installer-v2.7.2b.exe
+  Architecture: x64
+  InstallerType: inno
+  InstallerSha256: c6be3de65f63226d7da3e794300209be8b550c20c540ef04032106e7de607ff1

```

```suggestion
  InstallerSha256: c6be3de65f63226d7da3e794300209be8b550c20c540ef04032106e7de607ff1
  AppsAndFeaturesEntries:
  - DisplayName: PELD version 1.0
    DisplayVersion: 1.0
    Publisher:
    ProductCode: PELD_is1
```
Since the PackageVersion and Publisher are different than what is written to control panel, AppsAndFeaturesEntries are needed